doc/userguide/partials/options.rst

```diff
@@ -54,6 +54,12 @@
    continuously feed files to a directory and have them cleaned up when done. If
    this option is not set, pcap files will not be deleted after processing.
 
+.. option:: --pcap-file-directory-depth
+   Used with the -r option when the path provided is a directory.  This option
+   sets the maximum depth of the directory traversal (between 0 and 255).  
+   The default value is 0. Symlinks are ignored. 
```

Comment:
Does this mean that by default we're not doing any recursion into sub directories?

Reply:
Correct.  I thought it best if no options are provided there will be no change in behavior from the current release.

Reply:
Not sure yet what I would like the default to be. @jasonish any thoughts?

For now lets continue with the default behavior has you defined it.

Reply:
@jasonish I'll run with that.  I like the idea of an "enabled" or "disabled" flag.  It will be far clearer to users.